packages/core/src/workspace/schema/GraphqlEntitySchema.ts

```diff
@@ -2,7 +2,7 @@ import { t, type YasumuSchemaParsableToType } from '@yasumu/schema';
 import { WorkspaceModuleType } from '../modules/index.js';
 
 export const GraphqlEntitySchema = t.script({
-  annotation: WorkspaceModuleType.GraphQL,
+  annotation: WorkspaceModuleType && WorkspaceModuleType.GraphQL ? WorkspaceModuleType.GraphQL : 'GrapgQL',
```

Comment:
This shouldn't happen, let me see why it's happening. Could be circular dependency issue. In the meantime could you revert this